public/static/docs/command-reference/import.md

```diff
@@ -15,20 +15,21 @@ import.
 usage: dvc import [-h] [-q | -v] [-o [OUT]] [--rev [REV]] url path
 
 positional arguments:
-  url         URL of Git repository with DVC project to download from.
-  path        Path to data within DVC repository.
+  url         Location of DVC project or Git repository to download from
+  path        Path to a file or directory within the project or repository
+
 ```
 
 ## Description
 
 DVC provides an easy way to reuse datasets, intermediate results, ML models, or
-other files and directories tracked in another <abbr>DVC repository</abbr> into
-the workspace. The `dvc import` command downloads such a <abbr>data
-artifact</abbr> in a way that it is tracked with DVC, so it can be updated when
-the data source changes. (See `dvc update`.)
+other files and directories tracked in another Git/DVC repository into the
+workspace. The `dvc import` command downloads such a <abbr>data artifact</abbr>
+in a way that it is tracked with DVC, so it can be updated when the data source
+changes. (See `dvc update`.)
```

Comment:
This whole paragraph is unclear now that `dvc import` supports Git repos. Just adding "Git/" probably isn't enough. If we need to write a second paragraph for the Git repo case that's one option, or making this one clear enough if possible. In my experience, sometimes doc changes require more thought and time than the actual code changes.

Do you need a specific suggestion? I can help if needed. Hint: I would try to keep it as similar as possible (almost copy paste) of the latest intro paragraph in `dvc get`.